### PR TITLE
Fix make set-versions target 

### PR DESCRIPTION
**Pull request by daw3rd**
## Why are these changes needed?
Update .make.defaults and .make.transforms to address issues found in #541

### CODE REVIEW

File: .make.defaults [hunk 1]
```diff
@@ -578,12 +578,12 @@ MINIO_ADMIN_PWD= localminiosecretkey
 	@# Help: Update pyproject.toml to depend on lib versions defined in .make.versions 
 	@if [ -e pyproject.toml ]; then					\
 	    cat pyproject.toml | sed 									\
-		-e 's/"data-prep-toolkit-ray\(..\).*",/"data-prep-toolkit-ray\1$(DPK_LIB_VERSION)",/' 	\
-		-e 's/"data-prep-toolkit-spark\(..\).*",/"data-prep-toolkit-spark\1$(DPK_LIB_VERSION)",/' 	\
-		-e 's/"data-prep-toolkit-kfp\([=><][=><]\).*",/"data-prep-toolkit-kfp\1$(DPK_LIB_KFP_VERSION)",/' 	\
-		-e 's/"data-prep-toolkit\([=><][=><]\).*",/"data-prep-toolkit\1$(DPK_LIB_VERSION)",/' 		\
-		-e 's/"ray\[default\]\([=><][=><]\).*",/"ray\[default\]\1$(RAY)",/'				\
-		-e 's/"data-prep-toolkit-kfp-shared\(..\).*",/"data-prep-toolkit-kfp-shared\1$(DPK_LIB_KFP_VERSION)",/'     \
+		-e 's/"data-prep-toolkit-ray\([=><][=><]\).*"/"data-prep-toolkit-ray\1$(DPK_LIB_VERSION)"/' 	\
```
Comment (roytman) [inline .make.defaults]: if you want to narrow possible options, I'd define it as  `[=><~][=].

File: transforms/.make.transforms [hunk 2]
```diff
@@ -335,8 +335,9 @@ minio-stop:
 	if [ -e pyproject.toml ]; then	\
 	    dash_name=$$(echo $(TRANSFORM_NAME) | sed -e 's/_/-/g');		\
 	    cat pyproject.toml | sed 	\
-	    -e 's/"dpk-$(TRANSFORM_NAME)-transform-python\([=<>][=<>]\).*",/"dpk-$(TRANSFORM_NAME)-transform-python\1$(TRANSFORM_PYTHON_VERSION)",/'	\
-	    -e 's/"dpk-'$${dash_name}'-transform-python\([=<>][=<>]\).*",/"dpk-'$${dash_name}'-transform-python\1$(TRANSFORM_PYTHON_VERSION)",/'	\
+	    -e 's/"dpk_$(TRANSFORM_NAME)_transform_python\([=<>][=<>]\).*"/"dpk_$(TRANSFORM_NAME)_transform_python\1$(TRANSFORM_PYTHON_VERSION)"/'	\
```
Comment (roytman) [inline transforms/.make.transforms]: why not rename ones all dpk-...-transform-pyton ?

File: .make.defaults (hunk 1 above)
Comment (daw3rd) [reply .make.defaults]: Thanks and done.

File: transforms/.make.transforms (hunk 2 above)
Comment (daw3rd) [reply transforms/.make.transforms]: Thanks and done.

**Review by roytman — APPROVED** [review]

**Pull request merged**